docs_app/content/tutorial/7-flattening-operations.md

```diff
@@ -0,0 +1,132 @@
+# 7. Flattening Operations
+
+## TLDR:
+
+- "Flattening" is unwrapping nested (inner) Observables. (`Observable<Observable<T>> -> Observable<T>`)
+- There are four major flattening operations in RxJS: 
+    - **concat**: [`concatMap`](API), [`concatAll`](API), [`concat`](API), and [`concatWith`](API). Flattening one inner observable at a time, in order, without skipping any.
```

Comment:
Are `concat` and `merge` flattening operators? I thought only their concatXXX versions were flattening?